testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -423,6 +424,33 @@ public void testTrinoShowingSparkCreatedTables()
         onTrino().executeQuery("DROP TABLE " + trinoTableName(trinoTable));
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testPrestoWritingDataWithObjectStorageLocationProvider()
+    {
+        String baseTableName = "test_object_storage_location_provider";
+        String sparkTableName = sparkTableName(baseTableName);
+        String trinoTableName = trinoTableName(baseTableName);
+        String dataPath = "hdfs://hadoop-master:9000/user/hive/warehouse/test_object_storage_location_provider/obj-data";
+
+        String sparkTableDefinition = "CREATE TABLE %s (_string STRING, _bigint BIGINT) USING ICEBERG OPTIONS (" +
+                "'write.object-storage.enabled'=true," +
+                "'write.object-storage.path'='" + dataPath + "')";
+        onSpark().executeQuery(format(sparkTableDefinition, sparkTableName));
```

Comment:
inline `sparkTableDefinition` variable, it's redundant

(i know it's a copy of existing test code; i am doing this cleanup for existing tests in https://github.com/trinodb/trino/pull/8751/commits/566d41ae57b8836fd64e012c54318514d7b07bae)